### PR TITLE
Add support for PseudoDojo v0.5 `standard`/`stringent`

### LINKED ISSUE #24: Provide support for PseudoDojo v0.5 (standard+stringent)
The `get_pseudo_orbitals()` method:

https://github.com/aiidateam/aiida-wannier90-workflows/blob/aeceb3519ffd1cd071d5b98f81888052fff58163/src/aiida_wannier90_workflows/utils/pseudo/__init__.py#L54-L60

Currently only provides support for PseudoDojo v0.4 "standard" pseudopotentials. It would be useful to extend support to v0.5 as well, both the "standard" and "stringent" protocol.

### LINKED ISSUE #36: Add full support for PseudoDojo 0.5
We had partial support for `PseudoDojo/0.5/PBE/SR/standard/upf` and `PseudoDojo/0.5/PBE/SR/stringent/upf` in #34, would be great if those two jsons can be extended to cover all pseudos in the future.

### PR DESCRIPTION
**Pull request by mbercx**
Fixes #24 
Fixes #36

### CODE REVIEW

**Comment by mbercx** [comment]
@qiaojunfeng I think I got most/all of the new pseudopotentials of v0.5 of the PseudoDojo. To be honest I only made changes when I failed to submit, so there might still be some missing. ^^

Have a look to see if my choices for `pswfcs` and `semicores` seem sensible to you. So far most of our wannierizations looked pretty solid.

There is also a change for the NSCF step, setting `calculation` from `nscf` to `bands`. I forgot the motivation, to be honest. Was it because we provide the k-points as a list instead of a mesh? 🤔

File: src/aiida_wannier90_workflows/utils/workflows/builder/generator/__init__.py [hunk 1]
```diff
@@ -133,7 +133,15 @@ def get_nscf_builder(  # pylint: disable=too-many-arguments
 
     parameters["SYSTEM"]["nbnd"] = nbnd
 
+    # One can either use `nscf` mode or `bands` mode. For the `nscf` mode,
+    # the `nosym` and `noinv` must be set to True to avoid pw.x automatically
+    # adds more (symmetry-equivalent) kpoints to the user-specified kpoints.
+    # For the `bands` mode, pw.x will use exactly what the user provided.
```
Comment (mbercx) [inline src/aiida_wannier90_workflows/utils/workflows/builder/generator/__init__.py:139]: Why would one want to use`nscf` mode then?

Comment (mbercx) [reply src/aiida_wannier90_workflows/utils/workflows/builder/generator/__init__.py:139]: In that case, maybe we _don't_ want to make these changes here, but rather set these inputs in the protocol of the `EpwWorkChain` (once the overrides work as intended ^^).

Comment (qiaojunfeng) [reply src/aiida_wannier90_workflows/utils/workflows/builder/generator/__init__.py:139]: emm, indeed maybe that's a better choice. Then we have to first make the overrides working 😅

Comment (mbercx) [reply src/aiida_wannier90_workflows/utils/workflows/builder/generator/__init__.py:139]: I'll see if I can get that done. ^^

**Comment by mbercx** [comment]
@qiaojunfeng I bit the bullet and looked into a way of automatically generating the semicore JSON files. It can successfully regenerate the PseudoDojo JSON files you already supported, but obviously the heuristic for determining what states are semicore is _a bit_ constructed to achieve this.

Improvements most welcome! And kudos to @elinscott for the `upf_tools` package, helped me a bunch.

File: dev/test_pseudos.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+
+for family in (
+    "PseudoDojo/0.4/LDA/SR/standard/upf",
+    "PseudoDojo/0.4/LDA/SR/stringent/upf",
+    "PseudoDojo/0.4/PBE/SR/standard/upf",
+    "PseudoDojo/0.4/PBE/SR/stringent/upf",
+    "PseudoDojo/0.5/PBE/SR/standard/upf",
+    "PseudoDojo/0.5/PBE/SR/stringent/upf",
+    "PseudoDojo/0.5/PBE/SR/stringent/upf",
```
Comment (qiaojunfeng) [inline dev/test_pseudos.py:17]: seems one repeated line

**Comment by qiaojunfeng** [comment]
Thanks a lot!

**Pull request merged**